src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/PivotTests/UnoSamples_Pivot_Tests.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.PivotTests
 {
-	public class UnoSamples_Pivot_Tests : SampleControlUITestBase
+	public class PivotTests : SampleControlUITestBase
```

Comment:
Don't change this name, it conflicts with the namespace.

Reply:
I revert the change in my last commit